automation_tools/satellite6/upgrade/tasks.py

```diff
@@ -44,6 +45,7 @@
 
 
 logger = logger()
+data = {}
```

Comment:
This is initialize to create a empty dict, and then update it always. But yes, i will declare this locally into the function.